### DISCUSSION TITLE
error as bellow

### DISCUSSION THREAD

**Discussion by paragkulkarni** [l-lin/angular-datatables#1655] [Q&A]
Namespace '"C:/Users/parag/Documents/angular-project/zycus/node_modules/@angular/core/core"' has no exported member 'ɵɵNgModuleDeclaration'.

Install with this command - ng add angular-datatables

I am getting this compillation error on installation. Please give me some solution.

**Comment by shanmukhateja** [comment]
Did you run `npm install` and `npm run start` on your project? It sounds like Angular packages aren't compiled yet.

If it doesn't work, can you try deleting `node_modules` folder, run `npm install` then `npm run start`? After the project runs for first time, run `ng add` command again.

Please provide small reproducible StackBlitz link or Git repo to investigate this.
